### PR TITLE
docs(release): Add documentation and changelog changes

### PR DESCRIPTION
**Pull request by Abhinandan-Purkait**
### What does this PR do?
- Adds documentation for the newly added commands.
- Adds Changelog changes for new release.

### CODE REVIEW

**Review by vharsh — APPROVED** [review]
lgtm

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,45 @@
...
+* LocalPV-ZFS `volume` and `pools` listing commands.
+* BlockDevices listing by Nodes
+* PersistentVolumeClaim describe now supports jiva as well.
+
+## What's Next
+* Adding support for Mayastor
+* Adding support for managing Storage Devices
+* Adding support for performing sanity checks and flagging discrepancies like listing stale volumes or over-utilized pools.
+* Adding support for getting overall status (like kubectl openebs cluster-info)
+* Ability to generate raise GitHub issues with required troubleshooting information.
```
Comment (kmova) [inline CHANGELOG.md:22]:
```suggestion
* Ability to raise GitHub issues with required troubleshooting information.
```

File: README.md [hunk 2]
```diff
@@ -48,7 +57,7 @@ OpenEBSCTL is available on Linux, macOS and Windows platforms.
    cd kubectl-openebs_v0.1.0_Linux_x86_64
    sudo mv kubectl-openebs /usr/local/bin/
```
Comment (kmova) [inline README.md:58]: n_: update to latest version

Comment (Abhinandan-Purkait) [reply README.md:58]: Changed to `v0.2.0`

**Review by kmova — APPROVED** [review]

**Pull request merged**